libs/user-identity-service/src/test/java/uk/gov/di/ipv/core/library/useridentity/service/UserIdentityServiceTest.java

```
@@ -294,13 +294,34 @@ void areVCsCorrelatedReturnFalseWhenNameDifferentForBavCRI() throws Exception {
                                 USER_ID_1,
                                 BAV,
                                 createCredentialWithNameAndBirthDate(
-                                        "Jimmy", "Jones",
-                                        ""))); // BAV cri doesn't provide birthdate
+                                        "Jimmy", "Jones", "1000-01-01")));
```

This test was passing before since it was checking name correlation first and returned false when it had failed. Is there a reason we want to prioritise name correlation failures over a failed dob check? I'd expect this to throw since the VC would be invalid since it's an identity check credential with a missing DOB

---

Given we have a comment here saying that BAV doesn't provide DoBs should this be throwing an exception? Is the comment wrong, or are we OK with a missing DoB, or is it that the DoB field shouldn't exist at all in this case?

---

yeah I did wonder about that comment. I found the original PR which [ignores BAV for DOB correlation](https://github.com/govuk-one-login/ipv-core-back/pull/2736/changes#diff-03e309f6bbe7b1e939e8d7a3f5ad31be61b375cf9c88a8a05048bed7970df470) but [this PR](https://github.com/govuk-one-login/ipv-core-back/pull/2736/changes) much later on that removes it from the exclusion list so I assumed it was an old test 🤔 @blakeyp it looks like you put up the latter. Do you know if we should be excluding the BAV VCs for DOB correlation?

---

Ah so the original HMRC-backed BAV CRI didn’t give us DOB, but the Experian-backed version does (which is what actually went live). So the comment/test case is out of date, we should be checking it